common/src/main/java/com/discordsrv/common/config/main/AvatarProviderConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.discordsrv.common.config.main;
+
+import com.discordsrv.common.config.annotation.Order;
+import com.discordsrv.common.config.annotation.Untranslated;
+import org.spongepowered.configurate.objectmapping.ConfigSerializable;
+import org.spongepowered.configurate.objectmapping.meta.Comment;
+
+@ConfigSerializable
+public class AvatarProviderConfig {
+
+    @Untranslated(Untranslated.Type.VALUE)
+    @Order(0)
```

Comment:
Orders aren't required here, the order the fields are in already makes them go in this order